crates/bevy_input/src/lib.rs

```diff
@@ -50,9 +50,16 @@ impl Plugin for InputPlugin {
                 mouse_button_input_system.system(),
             )
             .add_event::<GamepadEvent>()
+            .init_resource::<GamepadProperty>()
             .init_resource::<Input<GamepadButton>>()
             .init_resource::<Axis<GamepadAxis>>()
             .init_resource::<Axis<GamepadButton>>()
+            .add_startup_system_to_stage(
+                bevy_app::startup_stage::POST_STARTUP,
+                gamepad_event_system.system(),
+            )
+            .add_system_to_stage(bevy_app::stage::EVENT_UPDATE, gamepad_event_system.system())
+            .init_resource::<Axis<GamepadButton>>()
```

Comment:
This is redundant. We already init this on line 56

Reply:
Removed